modules/aws-outposts-ingress-controller-fix-subnet.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-ingress-controller-fix-subnet_{context}"]
+= Ensuring the default Ingress Controller uses the correct subnet
```

Comment:
Should this chapter be placed before "Ensuring AWS Load Balancers use the correct subnet" chapter? It's more fundamental component of the cluster than the user's services which may or may not be created.